cmd/pipecd/server.go

```diff
@@ -65,6 +65,7 @@ var (
 
 const (
 	defaultPipedStatHashKey = "HASHKEY:PIPED:STATS"
+	defaultPipedStatTTL     = 2 * time.Minute
```

Comment:
This shouldn't be changeable even in the future; `pipedStatTTL` may be better to prevent us confusion.

Reply:
Nice point 👍 lets me adopt that 🙆‍♂️

Reply:
Done